gamelogic.js

```diff
@@ -1,17 +1,164 @@
 function displayWordSoFar(word, guesses) {
-  // WRITE ME
+    //  maak een array aan om de resultaten van de check in op te slaan
+    let resultArray = [];
+    //  boolean geeft aan of er een letter gevonden is,deze begint als false
+    let found = false;
+    //  for loop loopt over elke letter van het "gezochte"woord (word)
+    for (let i = 0; i < word.length; i++) {
+        //  for loop loopt over de guesses array en vergelijkt elke letter met elke letter van word
+        for (let j = 0; j < guesses.length; j++) {
+            //  als letter een match is,verander found in true; en plaatst de letter in de resultArray op de juiste index,daarna stopt deze loop;
+            if (guesses[j] == word.charAt(i)) {
+                found = true;
+                resultArray.push(word.charAt(i) + ' ')
+                break;
+                // niks gevonden? found is false,door met de volgende letter uit de guesses-array
+            } else {
+
+                found = false;
+            }
+        }
+        //  als er na alle letters uit de gues-array geen match is push dan een underscore met spatie in de resultArray
+        if (!found) {
+            resultArray.push('_ ');
+        }
+    }
+    // na het itereren return je de resultarray omgezet naar een string
+    return resultArray.join("");
 }
```

Comment:
Leuk om te zien dat je het op een andere manier hebt aangepakt als ikzelf. Prima code wel, heb hem gecloned en werkt perfect! Stukje geavanceerder dan mijn eigen oplossing. Super duidelijke comments, maakte het meteen super duidelijk om te lezen en begrijpen wat er gebeurde in de code.